x-pack/plugins/osquery/server/routes/pack/read_pack_route.ts

```diff
@@ -43,7 +43,14 @@ export const readPackRoute = (router: IRouter) => {
         body: {
           data: {
             ...rest,
-            ...attributes,
+            name: attributes.name,
```

Comment:
should `...rest` also not be spread here?

Reply:
Good question. I think that the only depdendency is on SO attributes, but for safety - I have also refactored ...rest :) Thanks @paul-tavares 👍